com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineUI.cs

```diff
@@ -238,7 +238,12 @@ static void Drawer_SectionCookies(SerializedHDRenderPipelineAsset serialized, Ed
 
         static void Drawer_SectionReflection(SerializedHDRenderPipelineAsset serialized, Editor owner)
         {
-            EditorGUILayout.PropertyField(serialized.renderPipelineSettings.supportSSR, Styles.supportSSRContent);
+            using (new EditorGUI.DisabledScope(!serialized.renderPipelineSettings.supportMotionVectors.boolValue))
+            {
+                EditorGUILayout.PropertyField(serialized.renderPipelineSettings.supportSSR, Styles.supportSSRContent);
+                serialized.renderPipelineSettings.supportSSR.boolValue &= serialized.renderPipelineSettings.supportMotionVectors.boolValue;
```

Comment:
The alternative is to let supportSSR = true in the asset, but sanitize the frame settings in FrameSettings::Sanitize.

Sanitizing the asset looks better to me because there are many places that query the supportSSR  from the asset (and not the frame settings).